client/src/RatingsAndReviews/Reviews/IndividualReviewTile.jsx

```diff
@@ -3,45 +3,87 @@ import PropTypes from 'prop-types';
-        <div className="irt-summary-overage">{truncateSummary(props.review.summary)[1]}</div>
-      </div>
-      <div className="irt-review-body">
-        <div className="irt-body-text">{props.review.body}</div>
-        <div className="irt-body-show"></div>
-        <div className="irt-photos">
-          {props.review.photos.map((photo, index) => {
-            return (
-              <img
-                className="irt-photo"
-                key={index} src={photo.url}
-                alt="image"
-                onClick={props.viewPhoto}/>
-            )
-          })}
-        </div>
-      </div>
-      <div className="irt-additional-info">
-        {createRecommendDiv(props.review.recommend)}
-        {createResponseDiv(props.review.response)}
-        <div className="irt-helpfulness-info">
-          <div className="irt-helpful-text">Helpful?</div>
-          <div className="irt-yes-clickable">Yes</div>
-          <div className="irt-rating-helpfulness">({props.review.helpfulness})</div>
-          <div className="irt-report-clickable">Report</div>
+class IndividualReviewTile extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      helpfulClicked: false,
+      reportClicked: false,
+      helpful: 0
+    }
+
+    this.handleReport = this.handleReport.bind(this);
+    this.handleHelpful = this.handleHelpful.bind(this);
+  }
+
+  handleHelpful() {
+    if (this.state.helpfulClicked) {
+      return
+    }
+
+    reviewHelpful(this.props.review.review_id);
+    this.setState({
+      helpfulClicked: true,
+      helpful: 1
+    });
+  }
+
+  handleReport() {
+    if (this.state.reportClicked) {
+      return
+    }
+
+    reviewReport(this.props.review.review_id);
+    this.setState({
+      reportClicked: true
+    });
+  }
+
+  render() {
+    if (this.state.reportClicked) {
+      return <div></div>
+    } else {
+      return (
+        <div className="irt">
+          <div className="irt-header">
+            <div className="irt-star-rating">{Stars(this.props.review.rating)}</div>
+            <div className="irt-username-and-date">✓ {this.props.review.reviewer_name}, {formatDate(this.props.review.date)}</div>
+          </div>
+          <div className="irt-review-summary">
+            <div className="irt-summary-trunc">{truncateSummary(this.props.review.summary)[0]}</div>
+            <div className="irt-summary-overage">{truncateSummary(this.props.review.summary)[1]}</div>
+          </div>
+          <div className="irt-review-body">
+            <div className="irt-body-text">{this.props.review.body}</div>
+            <div className="irt-body-show"></div>
+            <div className="irt-photos">
+              {this.props.review.photos.map((photo, index) => {
+                return (
+                  <img
+                    className="irt-photo"
+                    key={index} src={photo.url}
+                    alt="image"
```

Comment:
When we get to accessibility compliance, alt might need to be more descriptive-- but for now, surely fine